server/sse.go

```diff
@@ -40,7 +34,7 @@ type sseHandler struct {
 	config        config.Config
 }
 
-func newHandler(opt Option) sseHandler {
+func NewSSEHandler(opt Option) sseHandler {
```

Comment:
__[golint]__
* [golint] exported function NewSSEHandler should have comment or be unexported
* [golint] exported func NewSSEHandler returns unexported type server.sseHandler, which can be annoying to use

<!-- Comment by [SideCI](https://sideci.com) -->